### PR TITLE
Prepare for 7.8.0 Release

### PR DESCRIPTION
**Pull request by bperseghetti**
# 🎈 Release

Preparation for 7.8.0 release.

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [x] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by Pedro-Roque** [comment]
Adding #2431 to the list of PRs needing this update.

**Comment by Pedro-Roque** [comment]
@bperseghetti Awesome! I'll keep an eye on this.

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+### Gazebo Sim 7.8.0 (2024-07-22)
+
+1. Added support for spacecraft thrusters
+    * [Pull request #2431](https://github.com/gazebosim/gz-sim/pull/2431)
+
+1. Disable rendering tests that are failing on github actions
+    * [Pull request #2480](https://github.com/gazebosim/gz-sim/pull/2480)
+
+1. Merge from ign-gazebo6
+    * [Pull request #2477](https://github.com/gazebosim/gz-sim/pull/2477)
```
Comment (iche033) [inline Changelog.md:12]: I think we usually skip merge forward PRs in changelog.

**Review by iche033 — APPROVED** [review]
looks good to me

File: Changelog.md (hunk 1 above)
Comment (j-rivero) [reply Changelog.md:12]: I'll remove this in a subsequent commit.

**Comment by Pedro-Roque** [comment]
@bperseghetti when will the binaries for this be available on Ubuntu?

**Pull request merged**